Update features.html

jurjendevries opened the pull request:
Changed the Lightning description because Blue Wallet stopped providing this service.

ncoelho reviewed (changes requested):
No links to custodial services

jurjendevries commented:
> No links to custodial services

Hi @ncoelho , sorry I wasn't aware of that restriction. I've updated the file and also updated some other files I found related to Bluewallet Lightning. I hope everything looks fine now.

ncoelho commented:
The lightning page disappeared. It should not, the only thing that changed there is that the wallet does not suggest a default server anymore. 
Meaning people need to find a server or input their own.
